apps/backend/lib/graphql-bookshelf-bridge/fetcher.js

```diff
@@ -124,20 +124,35 @@ export default class Fetcher {
     return matches[0][0]
   }
 
-  _loadMany (relation, { method, loader, tap, fetchOpts }) {
-    return this.loaders.relations.load({relation, method})
-    .tap(tap)
```

Comment:
This `.tap` run was removed as I researched it and found nowhere where it was any longer used or ran. As a reminder, because I also had to look it up: .tap is a Bluebird Promise chain method that simply runs the passed function while passing along the results of the promise from before it to the next thing. In async/await language it is just running a function between two other functions. I would have just put it in here, but it was so clear that it wasn't in use I decided to make it go away.

Reply:
id love to move fully to async await and get rid of bluebird... in time

Reply:
for sure, you and I both